static/src/javascripts/projects/common/utils/geolocation.js

```diff
@@ -0,0 +1,31 @@
+define([
+    'Promise',
+    'common/utils/fetch-json',
+], function (
+    Promise,
+    fetch
+) {
+    var location;
+
+    return {
+        get: function() {
+            return new Promise(function(resolve, reject) {
+                if (location) return resolve(location);
+                else {
+                    fetch('https://api.nextgen.guardianapps.co.uk/geolocation', {
```

Comment:
that would be nice to use `config.page.ajaxUrl` so when we test locally it goes to localhost